prov/tcp/src/tcpx_conn_mgr.c

```diff
@@ -128,14 +129,23 @@ static int poll_fds_add_item(struct poll_fd_mgr *poll_mgr,
 				       util_ep.ep_fid.fid);
 		poll_mgr->poll_fds[poll_mgr->nfds].fd = tcpx_ep->conn_fd;
 		poll_mgr->poll_fds[poll_mgr->nfds].events = POLLOUT;
-
+		poll_mgr->poll_fds[poll_mgr->nfds].revents = 0;
+		break;
+	case REJECT_SOCK:
+		handle = container_of(poll_info->fid,
+					   struct tcpx_conn_handle,
+					   handle);
+		poll_mgr->poll_fds[poll_mgr->nfds].fd = handle->conn_fd;
+		poll_mgr->poll_fds[poll_mgr->nfds].events = POLLOUT;
+		poll_mgr->poll_fds[poll_mgr->nfds].revents = 0;
 		break;
 	case PASSIVE_SOCK:
 		tcpx_pep = container_of(poll_info->fid, struct tcpx_pep,
 					util_pep.pep_fid.fid);
 
 		poll_mgr->poll_fds[poll_mgr->nfds].fd = tcpx_pep->sock;
 		poll_mgr->poll_fds[poll_mgr->nfds].events = POLLIN;
+		poll_mgr->poll_fds[poll_mgr->nfds].revents = 0;
```

Comment:
revents = 0 is set in all possible cases (I think), just move outside switch statement